docs/usage/configuration-options.md

```diff
@@ -2392,8 +2394,11 @@ after 10pm and before 5:00am
 after 10pm and before 5am every weekday
 on friday and saturday
 every 3 months on the first day of the month
+* 0 2 * *
 ```
 
+Note: For Cron schedules, you _must_ use `*` for minutes, as Renovate doesn't support minute granularity.
```

Comment:
```suggestion
Note: For Cron schedules, you _must_ use the `*` wildcard for the minutes value, as Renovate doesn't support minute granularity.
```

The sentence we have now could probably be improved a bit, would this be better? It might help to explain that `*` is the wildcard property?

I'm not sure what the correct term is for a field/property/value in cron, so feel free to edit this suggestion so it uses the correct term... 😉